package.json

```diff
@@ -85,86 +86,90 @@
     "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.22.6",
+    "@babel/runtime": "^7.24.4",
     "@patternfly/patternfly": "^5.0.0",
     "@patternfly/react-core": "^5.0.0",
     "@patternfly/react-table": "^5.0.0",
-    "@redhat-cloud-services/frontend-components": "^4.0.0",
-    "@redhat-cloud-services/frontend-components-utilities": "^4.0.0",
-    "@unleash/proxy-client-react": "^3.5.0",
+    "@redhat-cloud-services/frontend-components": "^4.2.3",
+    "@redhat-cloud-services/frontend-components-utilities": "^4.0.4",
+    "@unleash/proxy-client-react": "^4.0.3",
```

Comment:
Seeing this error in the console `Unsatisfied version 3.6.0 from insights-chrome of shared singleton module @unleash/proxy-client-react (required ^4.0.3)`

Looks like maybe we can't update this yet

Reply:
I'm still seeing this warning with the new commits - unleash probably needs to stay where it was (3.6.0) because insights-chrome expects it